nibabies/workflows/bold/base.py

```diff
@@ -514,12 +511,16 @@ def init_func_preproc_wf(bold_file, has_fieldmap=False, existing_derivatives=Non
     )
     bold_confounds_wf.get_node("inputnode").inputs.t1_transform_flags = [False]
 
+    # Detect dummy scans
+    nss_detector = pe.Node(NonsteadyStatesDetector(), name='nss_detector')
+    nss_detector.inputs.in_file = ref_file
+
     # SLICE-TIME CORRECTION (or bypass) #############################################
     if run_stc:
         bold_stc_wf = init_bold_stc_wf(name="bold_stc_wf", metadata=metadata)
         # fmt:off
         workflow.connect([
-            (inputnode, bold_stc_wf, [('n_dummy_scans', 'inputnode.skip_vols')]),
+            (nss_detector, bold_stc_wf, [('n_dummy', 'inputnode.skip_vols')]),
```

Comment:
At least in fMRIPrep, we allow this to be set manually by setting `boldref_wf.inputnode.dummy_scans`. Is this not implemented in nibabies?

Reply:
Good catch - It is, I guess we should prioritize user input, and default to calculating it with the nonsteadystates detector otherwise.

Reply:
@effigies did the latest commit (7718b99) adequately address this?